app/assets/javascripts/student_profile/record_service.jsx

```diff
@@ -211,7 +211,8 @@
             datepickerOptions={{
               showOn: 'both',
               dateFormat: 'mm/dd/yy',
-              minDate: undefined
+              minDate: undefined,
+              maxDate: new Date
```

Comment:
btw @alexsoble this just set the maximum date, its straight from https://api.jqueryui.com/datepicker/#option-maxDate.